package.json

```diff
@@ -13,14 +13,16 @@
     "test": "test"
   },
   "devDependencies": {
+    "browserify": "^16.2.0",
     "eslint": "^1.10.3",
     "faucet": "0.0.1",
     "jsdom": "^3.1.2",
     "tape": "^4.2.2"
   },
   "scripts": {
     "test": "node test | faucet",
-    "posttest": "eslint *.js test/*.js test/**/*.js"
+    "posttest": "eslint *.js test/*.js test/**/*.js",
+    "build": "browserify index.js -o dist/bundle.js"
```

Comment:
@Livia-Santos Nice! You have a `script` :) I think it’s better if we name these build files what they really are, which is `dist/viewloader.js` that way we’re not creating problems down the track if we change that name in some way.